btrfs-progs.yaml

```diff
@@ -90,18 +90,12 @@ update:
 
 test:
   pipeline:
-    - runs: |
-        btrfs --version
-        btrfs-convert --help
-        btrfs-find-root --help
-        btrfs-image --help
-        btrfsck --help
-        btrfstune --help
-        fsck.btrfs --version
-        mkfs.btrfs --version
-        btrfs --help
-        fsck.btrfs --help
-        mkfs.btrfs --help
+    - uses: test/tw/ver-check
+      with:
+        bins: btrfs mkfs.btrfs
```

Comment:
What happened to fsck.btrfs? (I realize some of these don't actually return a version but it'd be good to know for sure!)

Reply:
It doesn't output anything useful with `-v`, `-V`, `--version`, just a usage message.